draft/2022-11-23-this-week-in-rust.md

```diff
@@ -250,18 +240,13 @@ Rusty Events between 2022-11-23 - 2022-12-21 🦀
     * [**Meetup: 25u16**](https://www.meetup.com/rust-saar/events/289663288/)
 * 2022-12-14 | Virtual (Boulder, CO, US) | [Boulder Elixir and Rust](https://www.meetup.com/boulder-elixir-rust/)
     * [**Monthly Meetup**](https://www.meetup.com/boulder-elixir-rust/events/zvxcsrydcqbsb/)
-
-### Asia
-
-* 2022-11-22 | Tokyo, JP | [Tokyo Rust Meetup](https://www.meetup.com/tokyo-rust-meetup)
-    * [**Fullstack Web Dev in Rust**](https://www.meetup.com/tokyo-rust-meetup/events/289661246/)
+* 2022-12-20 | Virtual (Washington, DC, US) | [Rust DC](https://www.meetup.com/rustdc//)
```

Comment:
Consider for 12/20: https://berline.rs/2022/12/20/rust-hack-and-learn.html

Reply:
Added!